### PR TITLE
feat: go-ipfs 0.8.0 + docs for repeated flags

### PR DESCRIPTION
**Pull request by aschmahmann**
Text still WIP, but wanted to start before I forgot. This explains how repeated flags are used in the `ipfs pin remote service` commands.

### CODE REVIEW

**Comment by lidel** [comment]
cc @jessicaschilling

File: markdown.go [hunk 1]
```diff
@@ -99,8 +99,17 @@ Flags are added through the query string. For example, the %s flag is the %s que
 }
 `+"```"+`
 
+Some flags may be repeated. For example, the --status flag may be reused as below:
```
Comment (jessicaschilling) [inline markdown.go:102]: `--status` in code syntax?

File: markdown.go [hunk 2]
```diff
@@ -99,8 +99,17 @@ Flags are added through the query string. For example, the %s flag is the %s que
...
+
+`+"```"+`
+> curl -X POST "http://127.0.0.1:5001/api/v0/pin/remote/service/ls?name=myservice&status=pinned&status=pinning"
+`+"```"+`
+
 ::: tip
 Some arguments may belong only to the CLI but appear here too. These usually belong to client-side processing of input, particularly in the `+"`add`"+` command.
+
+Additionally, as a convenince certain CLI commands may allow passing repeated flags as delimited lists such as
```
Comment (jessicaschilling) [inline markdown.go:111]:
```suggestion
Additionally, as a convenience certain CLI commands may allow passing repeated flags as delimited lists such as
```

Comment (Mr0grog) [inline markdown.go:111]: Slight clarification and typo fix here:

```suggestion
Additionally, as a convenience certain CLI commands may allow passing repeated flags as comma-delimited lists such as
```

File: markdown.go [hunk 3]
```diff
@@ -99,8 +99,17 @@ Flags are added through the query string. For example, the %s flag is the %s que
...
+`+"`ipfs pin remote service ls --status=pinned,pinning`"+`, however this does not apply to the HTTP API.
```
Comment (jessicaschilling) [inline markdown.go:112]:
```suggestion
`+"`ipfs pin remote service ls --status=pinned,pinning`"+`; however, this does not apply to the HTTP API.
```

**Review by jessicaschilling — COMMENTED** [review]
Added some extremely small nits — thanks @lidel for the heads-up.

File: markdown.go (hunk 2 above)
Comment (aschmahmann) [reply markdown.go]: The fact that the list is comma delimited is technically per-CLI command. i.e. if some command took lists of comma separated tuples like `cmd --tuples=(a,b) --tuples=(c,d)` the cmd command might decide on a different delimiter like `;` and so we'd be able to get `cmd --tuples=(a,b);(c,d)`.

Given that the only commands with delimiters use commas, I'd be ok with this change although we run the risk of accidentally not updating this if we add commands with different delimiters.

Comment (Mr0grog) [reply markdown.go]: Good point! I still think the language could be clarified a bit (so someone doesn’t assume an _implied_ “comma” there). Maybe:

> some CLI commands allow passing repeated flags as delimited lists (delimiters vary by command). For example: …

**Comment by lidel** [comment]
@aschmahmann should we merge this and regenerate https://docs.ipfs.io/reference/http/api/  ?
go-ipfs 0.8.0 already shipped :D

**Review by lidel — APPROVED** [review]
Switched to 0.8.0 and smoke-tested – LGTM.

@aschmahmann I propose we merge this, but re-generate https://docs.ipfs.io/reference/http/api/ after  fixes from https://github.com/ipfs/go-ipfs/pull/8010 are applied to avoid regressions in text quality.

**Comment by Stebalien** [comment]
@lidel I assume merging this was the right call? (or did we need to wait for a release)

**Pull request merged**